lib/matplotlib/backends/backend_ps.py

```diff
@@ -802,7 +802,7 @@ def draw_text(self, gc, x, y, s, prop, angle, ismath=False, mtext=None):
                 else:
                     kern = 0
                 lastgind = gind
-                thisx += kern/64.0
+                thisx -= kern/64.0
```

Comment:
This also may be an issue of me reading badly as this looked fine before the pep8 fixes